src/Tribe/Attendee_Registration/Modal.php

```diff
@@ -104,8 +111,8 @@ function append_modal_ar_template( $unused_content, $template_obj ) {
 			];
 		}
 
-		/** @var Tribe__Tickets__Attendee_Registration__View $view */
-		$view = tribe( 'tickets.attendee_registration.view' );
+		/** @var \Tribe\Tickets\Plus\Attendee_Registration\View $view */
+		$view = tribe( 'tickets-plus.attendee-registration.view' );
```

Comment:
:warning: Code after the RETURN statement on line 99 cannot be executed

<sub>hash: 308515509a92fe516454034f99cb2392</sub>